src/elements/content-sidebar/activity-feed/task-new/Task.scss

```diff
@@ -37,8 +37,13 @@ $bcs-task-left-offset: 10 * $bcs-task-grid + 2px;
     }
 }
 
-.bcs-Task-statusContainer {
-    margin-top: 2 * $bcs-task-grid;
+.bcs-Task-status {
+    margin-top: 3 * $bcs-task-grid;
+    margin-bottom: 2 * $bcs-task-grid;
+}
+
+.bcs-Task-dueDate {
+    margin-bottom: 12px;
```

Comment:
this is also on the grid, but we can do a sweep thru all of these to update them